components/cluster/command/root.go

```diff
@@ -93,6 +93,8 @@ func init() {
 			}
 
 			tidbSpec = spec.GetSpecManager()
+			deployer = deploy.NewDeployer("tidb", tidbSpec)
```

Comment:
ditto, something like `NewCommandSet` or `NewSubcommands` or so would be better IMO.

Reply:
what about put it in pkg/cluster as `Manager`?

Reply:
https://github.com/pingcap/tiup/pull/587/commits/db9083bee92acb9fac22975076962e62208e006b